typings/primordials.d.ts

```diff
@@ -106,12 +106,8 @@ declare namespace primordials {
   export const ReflectSet: typeof Reflect.set
   export const ReflectSetPrototypeOf: typeof Reflect.setPrototypeOf
   export import AggregateError = globalThis.AggregateError;
-  export const AggregateErrorLength: typeof AggregateError.length
-  export const AggregateErrorName: typeof AggregateError.name
```

Comment:
If the goal is to reduce the risk that contributors get confused by them, would it make more sense to `@deprecated` them rather than delete them?

```suggestion
  /** @deprecated This is not a useful primordial */
  export const AggregateErrorLength: typeof AggregateError.length
  /** @deprecated This is not a useful primordial */
  export const AggregateErrorName: typeof AggregateError.name
```

Reply:
My goal is a little bit of both actually. Adding deprecated results in lots of repetitive comments, and I personally prefer removing them, and re-adding them when needed.